Broader globalization culture configuration (#36247)
dotnet/AspNetCore.Docs: aspnetcore/blazor/globalization-localization.md

    .AddSupportedUICultures(new[] { "en-US", "es-CR" }));
```

> Supported cultures (<xref:Microsoft.AspNetCore.Builder.RequestLocalizationOptions.AddSupportedCultures%2A>) adds the set of the supported cultures for *globalization* (date, number, and currency formatting). Supported UI cultures (<xref:Microsoft.AspNetCore.Builder.RequestLocalizationOptions.AddSupportedUICultures%2A>) adds the set of the supported UI cultures *for localization* (translated UI strings for rendering content).

Author: It's a singular subject, so it gets the -s form of the verb. However, what I might do is just use the API as the subject. I'll take a look at this before merging.

In the preceding example, the same supported formatting cultures and UI cultures are specified in a narrow case where the app is only used in the United States and Costa Rica. Alternatively, an app can use a broader set of cultures for date, number, and currency formatting but only provide localized content for the United States and Costa Rica, as the following example demonstrates:

```csharp
var uiCultures = new[] { "en-US", "es-CR" };

var formattingCultures = CultureInfo
    .GetCultures(CultureTypes.SpecificCultures)
    .Select(c => c.Name)
    .ToArray();

var localizationOptions = new RequestLocalizationOptions()
    .SetDefaultCulture(uiCultures[0])
    .AddSupportedCultures(formattingCultures )
    .AddSupportedUICultures(uiCultures);

app.UseRequestLocalization(localizationOptions);
```

> In the preceding example, [`CultureTypes.SpecificCultures`](xref:System.Globalization.CultureTypes) returns only cultures that are specific to a country or region—such as `en-US` or `fr-FR`—which come with full, concrete globalization data (for dates, numbers, calendars, and other cultural UI) that .NET can use for accurate formatting and parsing. Neutral cultures, such as `en` or `fr`, don't always have complete globalization data and aren't included in this list.

Author: Spaces around em dashes aren't allowed by the style manual. I'll revert.

> In the preceding example, [`CultureTypes.SpecificCultures`](xref:System.Globalization.CultureTypes) returns only cultures that are specific to a country or region—such as `en-US` or `fr-FR`—which come with full, concrete globalization data (for dates, numbers, calendars, and other cultural UI) that .NET can use for accurate formatting and parsing. Neutral cultures, such as `en` or `fr`, don't always have complete globalization data and aren't included in this list.

Reviewer: ```suggestion
In the preceding example, [`CultureTypes.SpecificCultures`](xref:System.Globalization.CultureTypes) returns only cultures that are specific to a country or region—such as `en-US` or `fr-FR`—which come with full, concrete globalization data (for dates, numbers, calendars, and other cultural UI) that .NET can use for accurate formatting and parsing. Neutral cultures, such as `en` or `fr`, may not always have complete globalization data and are therefore not included in this list.
```
